recommenders/datasets/mock/movielens.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+Mock dataset schema to generate fake data for testing use. This will mimic the Movielens Dataset
+"""
+try:
+    import pandera as pa
+except ImportError:
+    raise ImportError("pandera is not installed. Try `pip install recommenders['dev']`")
```

Comment:
Is this clause to avoid breaking the code if people don't install dev?

Reply:
Yup.